pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -555,10 +549,13 @@ func (r Client) removePrecopies(precopies []planapi.Precopy, vmService *ovirtsdk
 	}
 }
 
-func eventFinishedSuccesfully(events []*ovirtsdk.Event) bool {
+func (r Client) isSnapshotRemovalFinished(events []*ovirtsdk.Event) bool {
 	for _, event := range events {
-		code := event.MustCode()
-		if code == REMOVE_SNAPSHOT_FINISHED_SUCCESS {
+		switch event.MustCode() {
```

Comment:
why not getting the code out and have error check instead of `MustCode()`?

Reply:
what do you mean by "getting code out"?

Reply:
code, err := event.Code()
if err != nil {
...
}
switch code {
...

Reply:
The `event.Code()` does not return an error but `(int64, bool)`, which is the code itself, and the second parameter checks, if it has a code or code is 0. In case we want to check the code is 0 and not the default golang value
from the go-ovirt
```go
func (p *Event) Code() (int64, bool) {
	if p.code != nil {
		return *p.code, true
	}
	var zero int64
	return zero, false
}
```